src/cli/index.js

```diff
@@ -362,7 +362,7 @@ config.init({
   binLinks: commander.binLinks,
   modulesFolder: commander.modulesFolder,
   globalFolder: commander.globalFolder,
-  cacheRootFolder: commander.cacheFolder,
+  cacheFolder: commander.cacheFolder,
```

Comment:
Yep the specific line where this is used is here: https://github.com/yarnpkg/yarn/blob/master/src/config.js#L241

In order to prevent BC break we decided to keep using `cacheFolder` in the public interface, but it seems I forgot one. Good catch @clanghout!